### ISSUE TITLE
font-family-name-quotes should ignore sass (scss) variable

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#745]
```scss
.text {
    color: $text-muted;
    font-family: $font-family-serif;
    font-size: $font-size-large;

    p {
        text-align: center;
    }
}
```
Output:
```
3:18	⚠  Expected double quotes around font-family name "$font-family-serif" (font-family-name-quotes) [stylelint]
```
